### PR TITLE
First setup and first userstory: "View tasks"

### PR DESCRIPTION
**Pull request by HishamAlmannai**

### CODE REVIEW

File: src/components/TaskList.spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import TaskList from './TaskList';
+
+describe('TaskList', () => {
+	it('renders a tasks and checkboxes', () => {
```
Comment (PReichetanz) [inline src/components/TaskList.spec.js:6]:
```suggestion
	it('renders tasks and five checkboxes', () => {
```

File: src/components/TaskList.spec.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+					{ id: 3, name: 'Each task should be visually separated', done: true },
+					{ id: 4, name: "Each task should display it's own checkbox", done: true },
+					{ id: 5, name: 'All tasks should be displayed in vertical order', done: true },
+				]}
+			/>
+		);
+
+		const litem = screen.getAllByRole('listitem');
+
+		expect(litem.length) === 5;
```
Comment (PReichetanz) [inline src/components/TaskList.spec.js:25]:
```suggestion
		expect(litem).toHaveLength(5);
```

**Review by PReichetanz — COMMENTED** [review]
Looks good, only two minor issues to think about 🚀

File: src/components/TaskList.spec.js (hunk 1 above)
Comment (HishamAlmannai) [reply src/components/TaskList.spec.js:6]: true

File: src/components/TaskList.spec.js (hunk 2 above)
Comment (HishamAlmannai) [reply src/components/TaskList.spec.js:25]: again what learned
thx

**Pull request merged**